integration-tests/tests/src/tests/list.ts

```diff
@@ -0,0 +1,103 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2023 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import { expect } from "chai";
+import { Realm } from "realm";
+
+import { openRealmBefore } from "../hooks";
+
+type Item<T = unknown> = { list: Realm.List<T> };
+
+describe.only("List", () => {
+  describe.only("with unconstrained (mixed) values", () => {
```

Comment:
Although convenient, I've seen green tests on CI where there were actually regressions on multiple occasions because someone merged a PR but had forgotten to remove their `.only` statements.

I'd suggest [using `--grep`](https://github.com/realm/realm-js/tree/bindgen/integration-tests#running-the-tests-during-development) instead, as that won't accidentally be committed.

Reply:
Makes sense! I can imagine it happening. 
If that's the case probably it will be worth adding a check on CI too, so it doesn't happen again 😁